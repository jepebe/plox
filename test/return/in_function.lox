fun f() {
  return "ok";
  print "bad";
  print "badder";
}

print f(); // expect: ok
