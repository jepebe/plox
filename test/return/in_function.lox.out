[38;5;226m[line 2] Warning after 'return': Unreachable code.[0m
0 error(s) and 1 warning(s) occurred
ok
